### PR TITLE
test(e2e): added the automation tests for error page

### PR DESCRIPTION
**Pull request by mudit94**
# Closes/Fixes/Resolves

### 

# Explain the feature/fix

test(e2e): added the automation tests for error page
## Screenshots

<details>
<summary>View Screenshots</summary>

image

</details>

### Ready-for-merge Checklist

- [x] Expected files: all files in this pull request are related to one feature request or issue (no stragglers)?
- [x] Does the change have appropriate unit tests?
- [x] Did tests pass?
- [x] Did you update or add any necessary documentation (README.md, WHY.md, etc.)?
- [x] Was this feature demoed and the design review approved?

### CODE REVIEW

File: e2e/integration/errorpagetests.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        cy.get( '#banner-links' ).should( 'be.visible' );
+    } )
+    it( 'Check whether user group spa link is present and having correct redirection', () => {
+        cy.contains( 'User Group' ).should( 'have.attr', 'href', 'https://qa.one.redhat.com/user-groups' );
+    } )
+    it( 'Check whether Home Link is visible and having correct redirection', () => {
+        cy.contains( 'Home' ).should( 'be.visible' ).should( 'have.attr', 'href', '/' );
+    } )
+    it( 'Check Documentation Link is visible and having correct redirection ', () => {
+        cy.contains( 'Documentation' ).should( 'be.visible' ).should( 'have.attr', 'href', 'https://qa.one.redhat.com/get-started' );
```
Comment (riginoommen) [inline e2e/integration/errorpagetests.js:28]:
```suggestion
        cy.contains( 'Documentation' ).should( 'be.visible' ).should( 'have.attr', 'href', 'https://qa.one.redhat.com/get-started' );
```
use QA_HOST variable

File: e2e/integration/errorpagetests.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    })
+     it( 'Check Feedback link is visible and having correct redirection ', () => {
+        cy.contains( 'Feedback' ).should( 'be.visible' ).should( 'have.attr', 'href', 'https://qa.one.redhat.com/feedback' );
```
Comment (riginoommen) [inline e2e/integration/errorpagetests.js:31]: use QA_HOST variable

File: e2e/integration/errorpagetests.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+     } )
+     it( 'Check Notifications is visible and having correct redirection ', () => {
+        cy.contains( 'Notifications' ).should( 'be.visible' ).should( 'have.attr', 'href', 'https://qa.one.redhat.com/notifications' );
```
Comment (riginoommen) [inline e2e/integration/errorpagetests.js:34]: use QA_HOST variable

File: e2e/integration/errorpagetests.js [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+        cy.contains( 'Whoops! There is nothing here' ).should( 'be.visible' );
+    } )
+    it( 'Check the home page link is visible and redirecting to Home SPA', () => {
+        cy.contains('Go to homepage').should('be.visible').should('have.attr','href','/')
+    } )
+    it( 'Check whether the links to other spas are visible', () => {
+        cy.get( '#banner-links' ).should( 'be.visible' );
+    } )
+    it( 'Check whether user group spa link is present and having correct redirection', () => {
+        cy.contains( 'User Group' ).should( 'have.attr', 'href', 'https://qa.one.redhat.com/user-groups' );
```
Comment (riginoommen) [inline e2e/integration/errorpagetests.js:22]: use QA_HOST variable

**Review by riginoommen — CHANGES_REQUESTED** [review]

File: e2e/integration/errorpagetests.js (hunk 1 above)
Comment (mudit94) [reply e2e/integration/errorpagetests.js]: fixed

File: e2e/integration/errorpagetests.js (hunk 4 above)
Comment (mudit94) [reply e2e/integration/errorpagetests.js]: this has been fixed

File: e2e/integration/errorpagetests.js (hunk 2 above)
Comment (mudit94) [reply e2e/integration/errorpagetests.js]: fixed

**Review by riginoommen — APPROVED** [review]

**Review by hybridx — APPROVED** [review]
LGTM

**Review by riginoommen — APPROVED** [review]

**Pull request merged**